clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -657,6 +657,9 @@ DPCT_ENUM_OPTION(
             "enqueued_barriers",
             int(DPCPPExtensionsDefaultEnabled::ExtDE_EnqueueBarrier),
             "Disable the enqueued barriers extension.", false),
+        DPCT_OPTION_ENUM_VALUE(
+            "free-function-queries", int(DPCPPExtensionsDefaultEnabled::ExtDE_FreeQueries),
```

Comment:
Disable the free function query experimental extension that allows getting 'id',  'item', 'nd_item', 'group', and             "'sub_group' instances globally.